.github/workflows/build.yml

```diff
@@ -93,15 +93,27 @@ jobs:
 
   tests:
     needs: build
-    name: ${{ matrix.grunt-cmd }}
+    name: ${{ matrix.grunt-cmd }}-${{ matrix.suite ||'' }}
     runs-on: ubuntu-22.04
     env:
       NODE: 16.x
 
     strategy:
       fail-fast: false
       matrix:
-        grunt-cmd: ['ci-webdriver-default', 'ci-webdriver-standard', 'ci-e2e-integration', 'ci-webdriver-default-mobile']
+        grunt-cmd: ['ci-webdriver-standard', 'ci-e2e-integration', 'ci-webdriver-default-mobile']
+        suite: [all]
+        include:
+          - grunt-cmd: ci-webdriver-default
+            suite: core
+          - grunt-cmd: ci-webdriver-default
+            suite: data
+          - grunt-cmd: ci-webdriver-default
+            suite: data
```

Comment:
Hi @ngaruko 

Am I seeing this wrong or are we running the same suite twice, `data`, and never running `enketo`?

Reply:
Good catch @dianabarsan . Definitely some copy/paste gone wrong 😮